lib/lockdown/lockdown.js

```diff
@@ -0,0 +1,82 @@
+const LOCKDOWN_PORT = 62078;
```

Comment:
is it always the same?

Reply:
Yeap. This is a daemon always running on the phone